articles/libraries/machine-learning/design.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Quantum machine learning library
+author: geduardo
```

Comment:
Add a description meta tag between title and author.
description: Learn the basic concepts of designing circuit models for the quantum circuit centric classifier.